src/psyclone/psyir/symbols/symbol_table.py

```diff
@@ -282,8 +283,10 @@ def deep_copy(self):
         # Fix the container links for imported symbols
         for symbol in new_st.imported_symbols:
             name = symbol.interface.container_symbol.name
+            orig_name = symbol.interface.orig_name
```

Comment:
Please could you add/extend a test of the deep-copy functionality in `symbol_table_test.py`?

Reply:
Added it to the existing `test_deep_copy` test.